packages/graphql/tests/integration/issues/915.int.test.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Driver, int, isInt } from "neo4j-driver";
+import debug from "debug";
+import { generate } from "randomstring";
+import { graphql, GraphQLError, GraphQLScalarType, Kind, ValueNode } from "graphql";
+import { gql } from "apollo-server";
+import neo4j from "../neo4j";
+import { Neo4jGraphQL } from "../../../src/classes";
+
+debug.enable("@neo4j/graphql:*");
```

Comment:
**question (non-blocking)**: I'm probably missing something here yet why do we use/enable `debug` here?